### ISSUE TITLE
NPE on Mac OS from PR #696

### ISSUE THREAD

**Issue by JamzTheMan** [RPTools/maptool#763]
**Describe the bug**
04343f8 Causes NPE on Mac OS.

`private int dragThreshold =
      (int) Toolkit.getDefaultToolkit().getDesktopProperty("DnD.gestureMotionThreshold");`

java.lang.NullPointerException
	at net.rptools.maptool.client.ui.ToolbarPanel$OptionPanel.add(ToolbarPanel.java:463)
	at net.rptools.maptool.client.ui.ToolbarPanel.createPointerPanel(ToolbarPanel.java:260)
	at net.rptools.maptool.client.ui.ToolbarPanel.<init>(ToolbarPanel.java:83)

**To Reproduce**
1. Run build 1.5.5 or current develop branch on Mac OS

**Comment by JamzTheMan** [comment]
@pnichols04 This change broke MacOS build for this release. I tracked it to this line of code but haven't had time to find a fix atm.

**Comment by Phergus** [comment]
We use `Toolkit.getDefaultToolkit()` all over in the code but this is the only place we use `Toolkit.getDefaultToolkit().getDesktopProperty()`.

 From the JavaDocs it seems like we should be using getDragThreshold() instead.

**Comment by Phergus** [comment]
@JamzTheMan Try this instead:

```
  private final int dragThreshold =
      (int) DragSource.getDragThreshold();
```
Works the same on Windows but I can't test Mac.

**Comment by JamzTheMan** [comment]
Fixed by #765
